app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def money_amount(amount)
+    number_to_currency(amount, unit: 'Rp.')
+  end
+
+  def short_summary(words, length, link)
+    summary = words[0 .. length]
```

Comment:
I think it'd be better using truncate:

```ruby
"This book is amazing and cool".truncate(17, separator: ' ')
```

That will truncate per-word rather than per-character, and also will add "..."